hello.go

```diff
@@ -65,6 +96,8 @@ func main() {
 		Revision:     revision,
 		Project:      project,
 		ProjectFound: projectFound,
+		Region:       region,
+		RegionFound:  regionFound,
```

Comment:
It seems if you leave Region empty string it means RegionFound:false. It might be good to do so for brevity of sample code.

Reply:
Got rid of it.